src/session.ts

```diff
@@ -29,9 +29,9 @@ function createSession() {
       return
     }
 
-    // If no environment variables, fall back to netrc
-    const machines = await getNetrcCredentials()
-    const creds = getCredentialsForMachine(machines, machineName)
+    // If no environment variables, fall back to .storyblok/credentials.json
+    const machines = await getCredentials()
```

Comment:
in a clean environment, launching a `login` will fail because `getCredentials()` if it doesn't find the `credentials.json` throws a filesystem error (`handleFileSystemError`) that should be catch and handled here

Reply:
fixed